azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/RunMojo.java

```diff
@@ -30,7 +30,7 @@
     
     public static final String FUNC_HOST_START_CMD = "func host start";
     public static final String FUNC_HOST_START_WITH_DEBUG_CMD = "func host start --language-worker -- " +
-            "\"-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005\"";
+            "\"-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=%d\"";
```

Comment:
Debug options for java are specified with 
```
"-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=portNumber
```

This full string needs to be configurable not just the port as there might be other protocols that can be used for debugging.